crawlerr/services/rag_service.py

```
@@ -341,6 +586,86 @@ async def _rerank_results(
+                pairs.append([query, content])
+            
+            # Get reranking scores from Qwen3 model
+            # Process pairs individually to avoid padding issues
+            import asyncio
+            loop = asyncio.get_event_loop()
+            
+            scores = []
+            for pair in pairs:
+                score = await loop.run_in_executor(
+                    None, 
+                    lambda p=pair: self.reranker.predict([p])[0]
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)